code/frameworks/angular/template/stories_angular-cli-default-ts/signal/button.component.ts

```diff
@@ -40,8 +40,7 @@ export default class SignalButtonComponent {
   /**
    * Optional click handler
    */
-  @Output()
-  onClick = new EventEmitter<Event>();
```

Comment:
should there still be a "test" for EventEmitter maybe? Or is that obsolete now?

Reply:
This component was newly created only to test signals. There are other components, which still uses `@Output` and are therefore well tested.